src/wp-includes/default-filters.php

```diff
@@ -749,4 +749,8 @@
 // Font management.
 add_action( 'wp_head', 'wp_print_font_faces', 50 );
 
+// Font library filters.
```

Comment:
I'd suggest moving these new filters up to be part of the Font management group of hooks.

```suggestion
```